### PR TITLE
feat: Append and edit env values

### PR DESCRIPTION
**Pull request by sbulav**
This PR adds following functionality:

- append environment name to buffer( default behavior `<cr>`)
- append environment value to buffer( mapping `<c-a>`)
- edit environment value for the current session( mapping `<c-e>`), addressing https://github.com/LinArcX/telescope-env.nvim/issues/2
- fixes weird behavior (bug?) with multiline values

I've set default behavior to env name as in my experience you often need a name for all kinds of scripts. I've also formatted the code with StyLua.

### CODE REVIEW

File: lua/telescope/_extensions/env.lua [hunk 1]
```diff
@@ -1,82 +1,118 @@
...
-        return
-        {
-          value = entry,
-          display =  make_display,
-          ordinal = string.format('%s %s', entry[1], entry[2])
-        }
-      end,
+				local function make_display(ent)
+					-- Concatinate multiline env values
+					local concat_value = entry[2]:gsub("\r?\n", " ")
```
Comment (LinArcX) [inline lua/telescope/_extensions/env.lua]: > fixes weird behavior (bug?) with multiline values

This line fixed this issue?

I don't see any difference in final output between my implementation and yours. Can you please elaborate more on this?

File: lua/telescope/_extensions/env.lua [hunk 2]
```diff
@@ -1,82 +1,118 @@
...
-        local tel_win_width
-        if width > 1 then
-          tel_win_width = width
-        else
-          tel_win_width = math.floor(cols * width)
-        end
-        local desc_width = math.floor(cols * 0.05)
-        local command_width = 22
+local function appendEnvironmentValue(prompt_bufnr)
+	local selection = action_state.get_selected_entry(prompt_bufnr)
```
Comment (LinArcX) [inline lua/telescope/_extensions/env.lua]: `action_state.get_selected_entry()` doesn't have any argument.

File: lua/telescope/_extensions/env.lua [hunk 3]
```diff
@@ -1,82 +1,118 @@
...
+	if selection.value == "" then
+		return
+	end
+	if vim.api.nvim_buf_get_option(vim.api.nvim_get_current_buf(), "modifiable") then
+		vim.api.nvim_put({ selection.value[2] }, "b", true, true)
+	end
+end
+
+local function appendEnvironmentName(prompt_bufnr)
+	local selection = action_state.get_selected_entry(prompt_bufnr)
```
Comment (LinArcX) [inline lua/telescope/_extensions/env.lua]: `action_state.get_selected_entry()` doesn't have any argument.

File: lua/telescope/_extensions/env.lua [hunk 4]
```diff
@@ -1,82 +1,118 @@
...
+	if selection.value == "" then
+		return
+	end
+	if vim.api.nvim_buf_get_option(vim.api.nvim_get_current_buf(), "modifiable") then
+		vim.api.nvim_put({ selection.value[1] }, "b", true, true)
+	end
+end
+
+local function editEnvironmentValue(prompt_bufnr)
+	local selection = action_state.get_selected_entry(prompt_bufnr)
```
Comment (LinArcX) [inline lua/telescope/_extensions/env.lua]: `action_state.get_selected_entry()` doesn't have any argument.

**Comment by LinArcX** [comment]
Looks overally good to me.

Also I added [contribution](https://github.com/LinArcX/telescope-env.nvim#contribution) to README. please use `.stylua.toml` provided in the repo.

I appreciate your effort and time for this PR.

File: lua/telescope/_extensions/env.lua (hunk 1 above)
Comment (sbulav) [reply lua/telescope/_extensions/env.lua]: Sure.
I use fish(not sure if the same bug exists on bash) and when the multiline environmental variable is defined, i.e.:
```
FZF_CTRL_T_COMMAND = fd --type file \
  --exclude ".git" \
  --exclude "venv*" \
  --exclude "*node_modules*" \
  --exclude "*ctags*" \
  --exclude "Library" \
  --exclude "Music" 
```

`C-n` and `C-p` shortcuts sometimes are not working, not all entries are displayed. See this demo:

https://user-images.githubusercontent.com/28604639/149325871-391b4c17-2d2b-44a8-a630-ccaff22b278e.mov

With concatenated entries problem is gone

File: lua/telescope/_extensions/env.lua (hunk 2 above)
Comment (sbulav) [reply lua/telescope/_extensions/env.lua]: fixed

File: lua/telescope/_extensions/env.lua (hunk 4 above)
Comment (sbulav) [reply lua/telescope/_extensions/env.lua]: fixed

File: lua/telescope/_extensions/env.lua (hunk 3 above)
Comment (sbulav) [reply lua/telescope/_extensions/env.lua]: fixed

File: lua/telescope/_extensions/env.lua (hunk 1 above)
Comment (LinArcX) [reply lua/telescope/_extensions/env.lua]: Thank you.

**Pull request merged**